### ISSUE TITLE
libpcl-1.7-all-dev not installable in Saucy

### ISSUE THREAD

**Issue by dirk-thomas** [ros-infrastructure/reprepro-updater#14]
http://ec2-54-67-30-249.us-west-1.compute.amazonaws.com:8080/view/IrelUbu/job/IrelUbu__pcl_conversions__ubuntu_saucy_amd64__binary/1/console

**Comment by tfoote** [comment]
I think this is a dupe of #13 libopenni-dev is a dependency of libpcl-apps-1.7-dev which is a dependency of libpcl-1.7-all-dev

**Comment by dirk-thomas** [comment]
Doesn't look like - the jobs are still failing for the same reason: http://54.183.26.131:8080/view/Ibin_uS64/job/Ibin_uS64__pcl_conversions__ubuntu_saucy_amd64__binary/3/console

**Comment by tfoote** [comment]
Verified in a saucy docker after 2nd fix for #13 

```
root@0b7110c6fddf:/# apt-get install libpcl-1.7-all-dev          
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following extra packages will be installed:
  adduser binutils build-essential bzip2 comerr-dev cpp cpp-4.8 dpkg-dev
[SNIP]
Setting up libpcl-apps-1.7-dev (1.7.1-3+saucy1) ...
Setting up libpcl-outofcore-1.7-dev (1.7.1-3+saucy1) ...
Setting up libpcl-people-1.7-dev (1.7.1-3+saucy1) ...
Setting up libpcl-1.7-all-dev (1.7.1-3+saucy1) ...
Processing triggers for libc-bin ...
Processing triggers for sgml-base ...
root@0b7110c6fddf:/# 
```
